src/components/submission-common.js

```diff
@@ -203,15 +203,16 @@ export const WithWarnings = props => {
     message
   } = props
 
-  const [iconShape, iconColor] = message ? Utils.switchCase(message.type,
+  const [iconShape, iconColor] = Utils.switchCase(message.type,
     ['error', () => ['error-standard', colors.warning()]],
     ['info', () => ['info-circle', colors.accent()]],
-    ['success', () => ['success-standard', colors.success()]]
-  ) : [undefined, undefined]
+    ['success', () => ['success-standard', colors.success()]],
+    ['none', () => [undefined, undefined]]
```

Comment:
What happens now if this _is_ undefined (or some other unexpected value)? Do we get a pageload error or does it show up as no content in the rendered page?